torch/autograd/gradcheck.py

```diff
@@ -225,7 +225,18 @@ def fail_test(msg):
             'gradcheck expects at least one input tensor to require gradient, '
             'but none of the them have requires_grad=True.')
 
-    output = _differentiable_outputs(func(*tupled_inputs))
+    func_out = func(*tupled_inputs)
+    output = _differentiable_outputs(func_out)
+
+    if not output:
+        for i, o in enumerate(func_out):
+            def fn(input):
+                return _as_tuple(func(*input))[i]
+            numerical = get_numerical_jacobian(fn, tupled_inputs, eps=eps)
+            for n in numerical:
+                if len(torch.nonzero(n)) > 0:
+                    fail_test('Numerical gradient for function expected to be zero')
```

Comment:
You should return here

Reply:
Could you also add a return in line 207? Thanks!

Reply:
Fixed